### PR TITLE
LINUX Env Changes and Update

### PR DESCRIPTION
**Pull request by AWADHAMBIKA**
Please review and comment

### CODE REVIEW

File: .Rhistory [hunk 1]
```diff
@@ -0,0 +1,79 @@
+getwd()
```
Comment (bimalkjha) [inline .Rhistory]: Please delete this unwanted file.

File: ChangeLog [hunk 2]
```diff
@@ -0,0 +1,656 @@
+Version: 1.3-15 (2017-04-13, released)
```
Comment (bimalkjha) [inline ChangeLog]: Delete this file too and any other unwanted files.

File: man/sqlTypeInfo.Rd [hunk 3]
```diff
@@ -0,0 +1,64 @@
+% file RODBC/man/sqlTypeInfo.Rd
+% copyright (C) 1999-2002  M. Lapsley
+% copyright (C) 2002-2013  B. D. Ripley
```
Comment (bimalkjha) [inline man/sqlTypeInfo.Rd:3]: If you have added any line to this file, good to add "copyright (c) 2020 Rocket Software" too.

**Review by bimalkjha — CHANGES_REQUESTED** [review]
Do we need to keep  RIBMDB_1.0-20.tar.gz on github? I think we need to remove it. Also, need to remove RODBC.o file. Good to rename RODBC.c to RIBMDB.c.

File: man/sqlTypeInfo.Rd (hunk 3 above)
Comment (AWADHAMBIKA) [reply man/sqlTypeInfo.Rd:3]: Done

File: .Rhistory (hunk 1 above)
Comment (AWADHAMBIKA) [reply .Rhistory]: Done

File: ChangeLog (hunk 2 above)
Comment (AWADHAMBIKA) [reply ChangeLog]: Done

**Pull request closed without merging**